packages/web/src/pages/sign-up-page/pages/CreateEmailPage.tsx

```diff
@@ -134,18 +160,22 @@ export const CreateEmailPage = () => {
               helperText={null}
             />
             <ErrorMessage name='email'>
-              {(errorMessage) => (
-                <Hint icon={IconError}>
-                  {errorMessage} {signInLink}
-                </Hint>
-              )}
+              {(errorMessage) =>
+                dirty ? (
```

Comment:
nice check on dirty here... so actually what is the situation, is the email carried over from sign-in page?